src/common/metric.tsx

```diff
@@ -15,14 +15,14 @@ import { formatDecimal, formatPercent } from 'common/utils';
 import { Metric } from 'common/metricEnum';
 
 export const ALL_METRICS = [
+  Metric.WEEKLY_CASES_PER_100K,
```

Comment:
Drive-by change to re-order the metrics in the compare tool so that the core metrics come first